src/OrbitQt/orbitlivefunctions.ui

```diff
@@ -54,6 +54,31 @@
         <string>Histogram</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
+        <item>
+         <widget class="QLabel" name="histogram_title_">
+          <property name="maximumSize">
```

Comment:
Is that needed?

Reply:
Yes, otherwise the label takes half of the vertical space.

Reply:
I was more wondering about the width value.

Reply:
It's default. Do you think I should just remove the line?

Reply:
Yep, just done that. The title has disappeared.

Reply:
Thanks for checking.